test/e2e/tests/request-queuing/dapp1-send-dapp2-signTypedData.spec.js

```diff
@@ -58,24 +57,13 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
         await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
         await driver.clickElement({
-          text: 'Next',
+          text: 'Connect',
           tag: 'button',
-          css: '[data-testid="page-container-footer-next"]',
         });
 
-        await driver.clickElement({
-          text: 'Confirm',
-          tag: 'button',
-          css: '[data-testid="page-container-footer-next"]',
-        });
-
-        await driver.waitUntilXWindowHandles(2);
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
-
-        // Open Dapp Two
+        // Open and connect to Dapp Two
```

Comment:
```suggestion
        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);

        // Open and connect to Dapp Two
```
we need to switch back to the test dapp to avoid context is discarded error

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/27561/commits/a6aee49a9adca593651c1a20072081d1579cbe57